scriptmodules/supplementary/sixaxis.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="sixaxis"
+rp_module_desc="Helper service to install & configure the latest drivers for official and third-party DualShock controllers (ps3controller replacement)\n\nNote: For Shanwan/GASIA third-party controllers, enable third-party support in the configuration options.\n\nTo pair controllers, use the RetroPie Bluetooth menu, choose 'Register and Connect...', then follow the on-screen instructions."
+rp_module_licence="GPL2 https://raw.githubusercontent.com/RetroPie/sixaxis/master/COPYING"
+rp_module_section="driver"
+
+function depends_sixaxis() {
+    local depends=(checkinstall libevdev-tools)
+
+    getDepends "${depends[@]}"
+
+    rp_callModule ps3controller remove
+}
+
+function sources_sixaxis() {
+    gitPullOrClone "$md_build/sixaxis" https://github.com/RetroPie/sixaxis.git
+}
+
+function build_sixaxis() {
+    cd sixaxis
+    make clean
+    make
+    md_ret_require+=("$md_build/sixaxis/bins/sixaxis-timeout")
```

Comment:
Not an issue but you don't have to use an array for a single file check. It's actually probably better to though - originally it only checked for single files and I think I extended it hence allowing either.